be/src/exec/vectorized/hash_join_node.cpp

```diff
@@ -406,6 +407,24 @@ Status HashJoinNode::close(RuntimeState* state) {
     return ExecNode::close(state);
 }
 
+bool _is_left_join(TJoinOp::type join_type) {
+    switch (join_type) {
+    case TJoinOp::LEFT_OUTER_JOIN:
+    case TJoinOp::LEFT_SEMI_JOIN:
```

Comment:
why left semi join and left anti join need not accumulate chunks?

Reply:
Oh, I'm wrong, only left outer join needn't accumulate chunks.